packages/shared/src/modules/onboarding/libs/enums/onboarding-validation-rule.enum.ts

```diff
@@ -0,0 +1,5 @@
+const OnboardingValidationRule = {
```

Comment:
Will we need more rules in the future?

Reply:
We will add some more if we need them in future.